### PR TITLE
Update Front Matter across SCBs and specific language in the Defender SCB to match OCC approved language 

### LINKED ISSUE #1393: Update SCBs with new front matter text approved by OCC
# 💡 Summary #

Update the Defender baseline front matter text with updated language and new general front matter language for all SCBs.

## Motivation and context ##

This work is needed to address the comments left by the OCC.

## Implementation notes ##

N/A

## Acceptance criteria ##

- [x] Defender SCB updated with approved OCC language
- [x] The following SCBs have updated front matter langauge 
  - [x] Entra ID / AAD
  - [x] Exchange
  - [x] Power BI
  - [x] Power Platform
  - [x] SharePoint & OneDrive
  - [x] Teams
  - [x] Removed Policies

### PR DESCRIPTION
**Pull request by ahuynhMITRE**
## 🗣 Description ##

OCC originally disapproved the consolidated baseline update document and SCBs for the Iceberg / Jellyfish release due to concerns that the SCBs may imply that CISA is telling the audience to upgrade their licenses to  higher tier. The updates found in this PR include SCB changes to the front matter to address this concern.

## 💭 Motivation and context ##

Change is required for the consolidated change document and the SCBs to be approved for the version 1.4.0 release.
Closes #1393 

## 🧪 Testing ##

N/A

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] PR targets the correct parent branch (e.g., main or release-name) for merge.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] Changes are sized such that they do not touch excessive number of files.
- [x] Related issues these changes resolve are linked preferably via [closing keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
- [x] All relevant type-of-change labels added.
- [x] All relevant project fields are set.
- [x] All relevant repo and/or project documentation updated to reflect these changes.
- [x] All relevant functional tests passed.
- [x] All automated checks (e.g., linting, static analysis, unit/smoke tests) passed.

## ✅ Pre-merge checklist ##

- [x] PR passed smoke test check.
- [x] Feature branch has been rebased against changes from parent branch, as needed

  Use `Rebase branch` button below or use [this](https://www.digitalocean.com/community/tutorials/how-to-rebase-and-update-a-pull-request) reference to rebase from the command line.
- [x] Resolved all merge conflicts on branch
- [x] Notified merge coordinator that PR is ready for merge via comment mention

## ✅ Post-merge checklist ##

- [ ] Feature branch deleted after merge to clean up repository.
- [ ] Verified that all checks pass on parent branch (e.g., main or release-name) after merge.

### CODE REVIEW

**Comment by ahuynhMITRE** [comment]
Note: The SCBs language changes are expected to be reviewed by the lead lawyer at OCC on November 4th then routed to EAD (front office) for final review. This SCB update cannot be merged into MAIN until all parties above approve the updates!

**Comment by ahuynhMITRE** [comment]
@mitchelbaker-cisa added in @adhilto's additional links and resources, updated the implementation steps for DEFENDER.6.3v1, and added his note on log retention in the license requirements sections for Defender and EXO. Please take a look at all of the above and make any changes. Once approved will add to the consolidated change document to get routed up to EAD.

File: PowerShell/ScubaGear/baselines/aad.md [hunk 1]
```diff
@@ -4,11 +4,11 @@
```
Comment (mitchelbaker-cisa) [inline PowerShell/ScubaGear/baselines/aad.md:4]: For[ policy group 7 license requirements](https://github.com/cisagov/ScubaGear/blob/1393-update-scbs-with-new-front-matter-text-approved-by-occ/PowerShell/ScubaGear/baselines/aad.md#license-requirements-6), I think we're missing the updated language: 

"Policies 7.4 through 7.9 require a Microsoft Entra ID P2 license; however, a third-party Privileged Access Management (PAM) solution may also be used to satisfy the requirements. If a third-party solution is used, then a P2 license is not required for the respective policies."

File: PowerShell/ScubaGear/baselines/teams.md [hunk 2]
```diff
@@ -3,12 +3,11 @@
```
Comment (mitchelbaker-cisa) [inline PowerShell/ScubaGear/baselines/teams.md:3]: For [Teams policy group 6 license requirements](https://github.com/cisagov/ScubaGear/blob/1393-update-scbs-with-new-front-matter-text-approved-by-occ/PowerShell/ScubaGear/baselines/teams.md#license-requirements-5), need to add the "However, this requirement can also be met through third-party services." (or similar language as you see fit) to indicate 3rd party service usage.

"DLP for Teams requires an E5 or G5 license. See [Microsoft Purview Data Loss Prevention: Data Loss Prevention for Teams | Microsoft Learn](https://learn.microsoft.com/en-us/office365/servicedescriptions/microsoft-365-service-descriptions/microsoft-365-tenantlevel-services-licensing-guidance/microsoft-365-security-compliance-licensing-guidance#microsoft-purview-data-loss-prevention-data-loss-prevention-dlp-for-teams) for more information. However, this requirement can also be met through third-party services."

File: PowerShell/ScubaGear/baselines/defender.md [hunk 3]
```diff
@@ -2,45 +2,19 @@
```
Comment (mitchelbaker-cisa) [inline PowerShell/ScubaGear/baselines/defender.md:2]: For [policy group 1](https://github.com/cisagov/ScubaGear/blob/1393-update-scbs-with-new-front-matter-text-approved-by-occ/PowerShell/ScubaGear/baselines/defender.md#license-requirements), need to add the "However, third-party solutions can be used to meet this requirement." (or similar language as you see fit) to indicate 3rd party service usage.

"Defender for Office 365 capabilities require Defender for Office 365 Plan 1 or 2. These are included with E5 and G5 and are available as add-ons for E3 and G3. However, third-party solutions can be used to meet this requirement."

Comment (mitchelbaker-cisa) [inline PowerShell/ScubaGear/baselines/defender.md:2]: For [policy group 4](https://github.com/cisagov/ScubaGear/blob/1393-update-scbs-with-new-front-matter-text-approved-by-occ/PowerShell/ScubaGear/baselines/defender.md#license-requirements-3), need to add the "However, third party services may be used to meet the DLP requirements." (or similar language as you see fit) to indicate 3rd party service usage.

**Review by mitchelbaker-cisa — CHANGES_REQUESTED** [review]
- AAD front matter looks good, left one comment for policy group 7 to verify license requirements language
- Exo looks good
- Teams front matter looks good, left one comment for policy group 6 to verify license requirements language
- Defender front matter looks good, left a couple comments for policy group 1 and 4 to verify license requirements language

File: PowerShell/ScubaGear/baselines/aad.md (hunk 1 above)
Comment (ahuynhMITRE) [reply PowerShell/ScubaGear/baselines/aad.md:4]: just updated!

File: PowerShell/ScubaGear/baselines/teams.md (hunk 2 above)
Comment (ahuynhMITRE) [reply PowerShell/ScubaGear/baselines/teams.md:3]: updated using similar language used for AAD in your first comment.

File: PowerShell/ScubaGear/baselines/defender.md (hunk 3 above)
Comment (ahuynhMITRE) [reply PowerShell/ScubaGear/baselines/defender.md:2]: added!

Comment (ahuynhMITRE) [reply PowerShell/ScubaGear/baselines/defender.md:2]: added!

**Review by mitchelbaker-cisa — APPROVED** [review]
Looks good, thanks for addressing those comments Andrew!

**Comment by ahuynhMITRE** [comment]
current status: routed back to CB Comms to go through final approvals on 11/7

**Comment by ahuynhMITRE** [comment]
@nanda-katikaneni & @schrolla Just got word from Chad that the SCBs have been approved! good to merge and add to 1.4.0 release!

**Pull request merged**